Composer/packages/server/src/workers/merge_worker.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Is this the file naming convention for worker files. We use the \*.worker.ts file naming pattern in other places.

Reply:
That naming convention exists for webpack loaders. I'm not sure it's worth snapping to that convention for node workers as it seems redundant.

`import { someThing } from '../workers/some-thing.worker'

Reply:
@a-b-r-o-w-n 

Then camelCase naming it \`import { someThing } from '../workers/mergeSchemas' seems a good option correct?

Reply:
Handled